src/libsyntax/parse/parser.rs

```diff
@@ -3778,9 +3771,6 @@ impl<'a> Parser<'a> {
 
     /// Parses an `if` or `if let` expression (`if` token already eaten).
```

Comment:
update this doc comment, `if let` is academic with `let` expressions

Reply:
Good catch! Fixed (will push later).